ftw/collectionblock/tests/test_add_collecionblock.py

```diff
@@ -153,6 +149,43 @@ def test_block_amount_limit(self, browser):
 
         browser.login().visit(self.page)
         self.assertEquals(
+            1,
+            len(browser.css('.even'))
+        )
+
+    @browsing
+    def test_is_table(self, browser):
+        create(Builder('sl content page').titled(u'Page 2'))
+        create(Builder('sl content page').titled(u'Page 3'))
+
+        create(Builder('sl collectionblock')
+               .titled(u'A collectionblock')
+               .within(self.page)
+               .having(block_amount=2)
+               .with_default_query())
+
+        browser.login().visit(self.page)
+        # check if there is a table
+        self.assertEquals(
+            1,
+            len(browser.xpath('//table[contains(@class, "sortable")]'))
+        )
+        # check if there is a table head and min. one element
+        self.assertLessEqual(
+            1,
+            len(browser.xpath('//thead//tr//td'))
```

Comment:
You could also just assert the content of the whole table with `browser.css('table.listing').first.lists` - this would be much more readable.
See https://ftwtestbrowser.readthedocs.io/en/stable/userdoc.html#tables